QUEST.tsv

```diff
@@ -2098,156 +2098,156 @@ QUEST_20150714_002097	But, Raius is the disciple I acknowledge.{nl}His passion t
+QUEST_20150714_002125	$I am confident in my knowledge of terms and theories more than the other disciples. {nl}That's why I studied magic. And while I was studying, I got to know the magic stone of interaction.
+QUEST_20150714_002126	$I came here when Helgasercle attacked Mage Tower. {nl}I thought that, if I could use the magic stone of interaction, I'd be able to use magic.
+QUEST_20150714_002127	$But, Akon chased me from Mage Tower. {nl}Due to this disturbance, the magic stone activated defensive magic and my and Akon's spirit were linked.
+QUEST_20150714_002128	$As a result, Akon was sealed and my body disappeared without a trace. {nl}We became shackled to each other.
+QUEST_20150714_002129	$I've been tied here for hundreds of years… {nl}Since I can't do anything with my astral body.
+QUEST_20150714_002130	$The bigger problem is that Akon was released as well. {nl}I lost the body, but Akon, who is a demon, is still alive.
+QUEST_20150714_002131	$Memorial Stone
+QUEST_20150714_002132	$By avoiding the eyes of Helgasercle, I was able to come here to continue my research. {nl}I needed to find a way: a way to reduce the power of demons and increase that magical power.
+QUEST_20150714_002133	$By my name, Balryus, the one who walks the way of Linker, I swear that I will not surrender to the demons.
+QUEST_20150714_002134	$I've made many magical tools, but I wasn't able to repel the demon attacks. {nl}Without checking out the result, I close my eyes.
+QUEST_20150714_002135	$The Records of Investigation
+QUEST_20150714_002136	$The basics of Link Magic are to find the common factors in each others' spirits. {nl}So it is impossible if the receiver does not possess any magical abilities or bases. {nl}
+QUEST_20150714_002137	$I've completed the first stage magic stone. {nl}It still does not satisfy the previous objective of delivering the perfect magic and magical power somewhat, {nl}but now at last it is possible to accept Link Magic even without magical ability.{nl}
+QUEST_20150714_002138	$Plan
+QUEST_20150714_002139	$If a magician finds a way to share all his abilities and knowledge to another, even a person without magical abilities can become a strong magician within a short period of time.
+QUEST_20150714_002140	$This way, you'd be able to hide from the eyes of Helgasercle who is extremely afraid of the disciple of Agailla Flurry appearing.
+QUEST_20150714_002141	$However, I'm not sure if this is at all possible or won't have any side effects.
+QUEST_20150714_002142	$Design Plan
+QUEST_20150714_002143	$The magic stone of the interaction can be worked on behalf of Link. {nl}However, maintaining the stone of interaction will require a considerable amount of magical power.
+QUEST_20150714_002144	$Hence, we decided to maintain the magic stone of the interaction with 12 regenerating magical power stones. {nl}If no problems arise, we will be able to maintain it, but if the magical power supply is cut, it won't stand for a long time.
```

Comment:
"Hence, we decided to maintain the magic stone of interaction with 12 regenerating magical power stones. {nl}If no problems arise, we will be able to maintain it, but if the magical power supply is cut, it won't last long."